### PR TITLE
site updates: fix some usage informations, make authorship clear, add logo

### LINKED ISSUE #208: Quesion about the appropriate tomograms for pytom picking
### Discussed in https://github.com/SBC-Utrecht/pytom-match-pick/discussions/206

<div type='discussions-op-text'>

<sup>Originally posted by **PeterVanF** August  6, 2024</sup>

Should add the information from my response to this thread to the documentation.

### PR DESCRIPTION
**Pull request by McHaillet**
Some updates for the site (see title).

Closes #208

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,23 +142,7 @@ Contributions to the project are very welcome! Feel free to make a pull request
...
-    VOLUME = {24},
-    YEAR = {2023},
-    NUMBER = {17},
-    ARTICLE-NUMBER = {13375},
-    URL = {https://www.mdpi.com/1422-0067/24/17/13375},
-    ISSN = {1422-0067},
-    DOI = {10.3390/ijms241713375}
-}
-```
+Chaillet, M. L., van der Schot, G., Gubins, I., Roet, S., Veltkamp, R. C., & Förster, F. (2023). Extensive angular sampling enables the sensitive localization of macromolecules in electron tomograms. _International Journal of Molecular Sciences_, 24(17), 13375. <https://doi.org/10.3390/ijms241713375>
```
Comment (sroet) [inline README.md:145]: We should probably add a [citation file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) as well

File: mkdocs.yml [hunk 2]
```diff
@@ -12,19 +12,20 @@ nav:
...
 theme:
+  logo: images/60S_masked.png
   name: material
   features:
     - navigation.instant
     # - navigation.instant.progress  # possible progress bar, but site is too small to justify
     - navigation.tracking
     - navigation.tabs
-    - navigation.tabs.sticky
+    # - navigation.tabs.sticky
```
Comment (sroet) [inline mkdocs.yml:28]: Just for my information: Why did you decide against sticky tabs?

**Review by sroet — APPROVED** [review]
1 question and a suggestion we can split out into a separate issue. 

LGTM, feel free to merge

File: README.md (hunk 1 above)
Comment (McHaillet) [reply README.md:145]: To the repository you mean right? Will make it an issue.

Comment (sroet) [reply README.md:145]: Yeah, I meant to the repository

File: mkdocs.yml (hunk 2 above)
Comment (McHaillet) [reply mkdocs.yml:28]: Good that you ask. I though it kept more info on screen on mobile, but I just double checked and mobile already has a separate interface. So i will switch back to sticky tabs

Comment (McHaillet) [reply mkdocs.yml:28]:
```suggestion
    - navigation.tabs.sticky
```

**Pull request merged**